### PR TITLE
Updating e2e script to run e2e test on arm64 machine

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Updating e2e script to run e2e test on arm64 machine

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - tested on louhi pipeline.
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

**Comment by ashmeenkaur** [comment]
please change Target to temp_sept_release

**Comment by Tulsishah** [comment]
> please change target to tmp_sept_release

Done

File: tools/cd_scripts/e2e_test.sh [hunk 1]
```diff
@@ -58,8 +58,8 @@ then
     sudo apt install -y fuse
 
     # download and install gcsfuse deb package
-    gsutil cp gs://gcsfuse-release-packages/v$(sed -n 1p details.txt)/gcsfuse_$(sed -n 1p details.txt)_amd64.deb .
-    sudo dpkg -i gcsfuse_$(sed -n 1p details.txt)_amd64.deb |& tee -a ~/logs.txt
+    gsutil cp gs://gcsfuse-release-packages/v$(sed -n 1p details.txt)/gcsfuse_$(sed -n 1p details.txt)_$(dpkg --print-architecture).deb .
```
Comment (sethiay) [inline tools/cd_scripts/e2e_test.sh:61]: define and use variables for repeated commands like $(dpkg --print-architecture) & $(uname -i)

File: tools/cd_scripts/e2e_test.sh [hunk 2]
```diff
@@ -49,6 +49,12 @@ echo User: $USER &>> ~/logs.txt
 echo Current Working Directory: $(pwd)  &>> ~/logs.txt
 
 # Based on the os type in detail.txt, run the following commands for setup
+
+architecture=$(dpkg --print-architecture)
+# architecture can be amd64 or arm64
```
Comment (sethiay) [inline tools/cd_scripts/e2e_test.sh:54]: please write comments above code line.

**Review by sethiay — DISMISSED** [review]

**Review by sethiay — APPROVED** [review]

**Pull request merged**